dart/lib/src/noop_client.dart

```diff
@@ -21,17 +22,7 @@ class NoOpSentryClient implements SentryClient {
   SentryOptions options;
 
   @override
-  String origin;
-
-  @override
-  List<int> bodyEncoder(
-    Map<String, dynamic> data,
-    Map<String, String> headers,
-  ) =>
-      [];
-
-  @override
-  Map<String, String> buildHeaders(String authHeader) => {};
+  Transport transport;
```

Comment:
why is this necessary? one should not care about the `transport` when implementing `SentryClient`